include/cutlass/epilogue/collective/xe_epilogue.hpp

```diff
@@ -310,12 +317,27 @@ class CollectiveEpilogue<
     auto sg_coord = make_coord(sg_m_coord, sg_n_coord, k_coord, l_coord);
 
     bool is_C_load_needed = is_source_supported && fusion_callbacks.is_C_load_needed();
+    
+    // Represent the full output tensor
+    Tensor mD_mnl = params.xe_store_d.get_pvc_tensor(make_shape(M,N,L));
+
+    // Tile the output tensor per CTA
+    Tensor g_cta_D_mnl = local_tile(mD_mnl, CtaTileMNK{}, make_coord(_,_,_), Step<_1,_1, X>{});             // (BLK_M,BLK_N,m,n,l)
+    
+    // Slice to get the tile this CTA is responsible for                                                 // (BLK_M,BLK_N)
+    Tensor g_cta_D = g_cta_D_mnl(_,_,m_coord,n_coord,l_coord);                                                   // (BLK_M,BLK_N)
```

Comment:
I am wondering here, if it should be possible to avoid this and have something like 
```C++
 Tensor g_cta_D_mnl  = local_tile(mD_mnl, CtaTileMNK{}, make_coord(m_coord,n_coord,l_coord), Step<_1,_1, X>{}); 
```

Reply:
It is possible if I manually construct the tile for MN dimesnions. I can only use the last argument(projection) if the previous two have the same number of modees.